servers/quarkus-common/src/main/java/org/projectnessie/quarkus/providers/storage/BigTableBackendBuilder.java

```diff
@@ -50,10 +50,14 @@ public class BigTableBackendBuilder implements BackendBuilder {
   public Backend buildBackend() {
     CredentialsProvider credentialsProvider = null;
     Exception googleCredentialsException = null;
-    try {
-      credentialsProvider = credentialsProviderInstance.get();
-    } catch (Exception e) {
-      googleCredentialsException = e;
+    // The disableCredentialsLookupForTests() is only used for tests and only to prevent the
+    // OpenTelemetry/global-tracer static-field initialization race described in #9866
+    if (!bigTableConfig.disableCredentialsLookupForTests()) {
```

Comment:
I'm not sure "ForTests" actually adds any guardrails, but it makes the java code look odd. Could you name the java method simply as `disableCredentialsLookup()`? I suppose end users will get enough warning from having "credentials" in the name.

Reply:
... and of course, we'll keep it undocumented (other than javadoc)

Reply:
IMHO, the code reads awkwardly, but up to you :)